x-pack/plugins/observability/public/typings/data_handler/index.d.ts

```diff
@@ -25,9 +33,49 @@ interface Series {
 interface FetchDataResponse {
   title: string;
   appLink: string;
-  stats: Stat[];
   series: Series[];
 }
+
+interface LogsFetchDataResponse extends FetchDataResponse {
+  stats: {
+    [logName: string]: Numeral;
+  };
+}
+
+interface MetricsFetchDataResponse extends FetchDataResponse {
+  stats: {
+    hots: Numeral & Color;
```

Comment:
I think it would be easier to read if you write it out instead of making a union. And I think `value` works better than `numeral`. The type already tells the consumer it must be a number.

```suggestion
    host: {
      value: number;
      color?: string;
    };
```

Reply:
Alternatively make distinct types:

```ts
type NumberStat = {
  value: number;
  color?: string
}

type PercentageStat = {
  pct: number;
  color?: string
}

type ByteStat = {
  bytes: number;
  color?: string
}
```